Detectors/EMCAL/base/include/EMCALBase/Constants.h

```diff
@@ -21,6 +23,12 @@ enum {
   EMCAL_LEDREFS = 24,   ///< Number of LEDs (reference/monitors) per module for EMCAL; one per StripModule
   EMCAL_TEMPSENSORS = 8 ///< Number Temperature sensors per module for EMCAL
 };
+
+namespace Constants {
```

Comment:
namespaces should be lowercase.

Reply:
... and I would question why having a namespace at all. Nothing will conflict with it and it's already pretty clear by being uppercase it's a constant (not to mention the fact you have a namespace EMCAL and then a variable `EMCAL_...`).